client/data/emails/use-create-titan-mailbox-mutation.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+import { useMutation } from 'react-query';
+import wpcom from 'calypso/lib/wp';
+
+/**
+ * Creates a mailbox for a Professional Email (Titan) account
+ *
+ * @param {string} domainName The domain name of the mailbox
+ * @param  {string} name The name of the user associated with the mailbox
+ * @param {string} mailboxName The mailbox name
+ * @param  {string} password The password to the mailbox
```

Comment:
Nit: it looks like some of these lines have additional whitespace after the `@param`.

Reply:
```suggestion
 * @param {string} name The name of the user associated with the mailbox
 * @param {string} mailboxName The mailbox name
 * @param {string} password The password to the mailbox
```